### PR TITLE
fix(stack-trace): Fix stack trace not properly styled

### PR DESCRIPTION
**Pull request by priscilawebdev**
follow-up of https://github.com/getsentry/sentry/pull/37545

### CODE REVIEW

**Review by narsaynorath — APPROVED** [review]
lgtm

File: static/app/components/events/interfaces/crashContent/stackTrace/contentV3.tsx [hunk 1]
```diff
@@ -209,29 +209,45 @@ function Content({
...
+      </Frames>
     </Wrapper>
   );
 }
 
 export default Content;
 
-const Wrapper = styled('ul')<{isHoverPreviewed?: boolean}>`
+const Wrapper = styled('div')`
+  && {
```
Comment (evanpurkhiser) [inline static/app/components/events/interfaces/crashContent/stackTrace/contentV3.tsx:243]: What does the double `&&` do again?

Comment (priscilawebdev) [reply static/app/components/events/interfaces/crashContent/stackTrace/contentV3.tsx:243]: it's like `!important` but more does not feel like 🙈

Comment (evanpurkhiser) [reply static/app/components/events/interfaces/crashContent/stackTrace/contentV3.tsx:243]: https://dev.to/magischerzwerg/double-ampersand-trick-in-sass-with-react-4khe ah it changes the specificity by including the class twice in the selector 🤔

**Pull request merged**